### PR TITLE
Python/sort/Sort_An_Arraysof0s1sand2s.py

### PR DESCRIPTION
**Pull request by soumikbaithalu**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

Yes I have read this.

### Description

0s 1s and 2s are segregated into ascending order

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

This is issue #4521

### CODE REVIEW

File: Python/sort/Sort_An_Arraysof0s1sand2s.py [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+    Total = listofZero+listofOne+listofTwo
+    return Total 
+
+
+
+
+arr = list(map(int,input("Enter the Number:\n").split()))
+n = len(arr)
+print(sort012(arr,n))
```
Comment (siddharth25pandey) [inline Python/sort/Sort_An_Arraysof0s1sand2s.py]: Display print message, what are you showing as output.

File: Python/sort/Sort_An_Arraysof0s1sand2s.py [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+'''
+Time Complexity is O(n)
+Space Complexity is O(1)
+
+Input: 
+N = 5
+arr[]= {0 2 1 2 0}
+
+Output:
+0 0 1 2 2
```
Comment (siddharth25pandey) [inline Python/sort/Sort_An_Arraysof0s1sand2s.py]: Program is not matching with Sample Input & Output
**Display message is not present in Sample part**

File: Python/README.md [hunk 3]
```diff
@@ -103,11 +103,13 @@
 - [House Robber](cp/house_robber.py)
 - [Number of K-Length Sublists with Average at Least Target](cp/Number_of_K-Length_Sublists_with_Average_at_Least_Target.py)
 - [Fence Painting Algorithm](cp/Paint_Fence.py)
+- [Dutch national flag problem](cp/Sort_An_Arraysof0s1sand2s.py)
 - [Find Intersection of Two Given Linked Lists](cp/find_intersection_of_two_linked_lists.py)
 - [Maximum value of each subarray of size k](cp/SlidingWindowMaximum.py)
 - [Count the number of anagrams](cp/CountAnagrams.py)
 - [Hailstone Sequence](cp/Hailstone_sequence.py)
 
+
```
Comment (siddharth25pandey) [inline Python/README.md:112]: Remove blank line

File: Python/README.md [hunk 4]
```diff
@@ -165,6 +167,8 @@
 - [Wiggle Sort](sort/Wiggle_sort.py)
 - [Counting Sort](sort/Counting_Sort.py)
 
+
+
```
Comment (siddharth25pandey) [inline Python/README.md]: Remove blank lines

File: Python/sort/Sort_An_Arraysof0s1sand2s.py [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+            listofOne.append(1)
+        else:
+            listofTwo.append(2)
+
+    Total = listofZero+listofOne+listofTwo
+    return Total 
+
+
+
+
```
Comment (siddharth25pandey) [inline Python/sort/Sort_An_Arraysof0s1sand2s.py]: Remove blank lines

File: Python/sort/Sort_An_Arraysof0s1sand2s.py [hunk 6]
```diff
@@ -0,0 +1,39 @@
...
+Input: 
+N = 5
+arr[]= {0 2 1 2 0}
+
+Output:
+0 0 1 2 2
+Explanation:
+0s 1s and 2s are segregated 
+into ascending order.
```
Comment (siddharth25pandey) [inline Python/sort/Sort_An_Arraysof0s1sand2s.py]: Remove this redundant part

**Review by siddharth25pandey — CHANGES_REQUESTED** [review]
@soumik2012 Do the changes as suggested

File: Python/sort/Sort_An_Arraysof0s1sand2s.py (hunk 5 above)
Comment (soumikbaithalu) [reply Python/sort/Sort_An_Arraysof0s1sand2s.py]: @siddharth25pandey as you suggested I modified the code and readme.md file

File: Python/sort/Sort_An_Arraysof0s1sand2s.py (hunk 1 above)
Comment (siddharth25pandey) [reply Python/sort/Sort_An_Arraysof0s1sand2s.py]: > Display print message, what are you showing as output.

@SKAUL05 Not resolved yet

Comment (soumikbaithalu) [reply Python/sort/Sort_An_Arraysof0s1sand2s.py]: is resolved now @siddharth25pandey

**Review by siddharth25pandey — APPROVED** [review]
@soumik2012 LGTM 💯 Able to merge

**Comment by siddharth25pandey** [comment]
@all-contributors please add @soumik2012 for his code

**Comment by soumikbaithalu** [comment]
@siddharth25pandey one more favor please could you reassigned to me

**Comment by siddharth25pandey** [comment]
> @siddharth25pandey one more favor please could you reassigned to me

@soumik2012 It was assigned earlier to you. But you had removed from your choice by own. 
Anyways, I'm going to assign you, but Please give a brief explanation why did you do so ?

**Comment by soumikbaithalu** [comment]
> > @siddharth25pandey one more favor please could you reassigned to me
> 
> @soumik2012 It was assigned earlier to you. But you had removed from your choice by own.
> Anyways, I'm going to assign you, but Please give a brief explanation why did you do so ?

Actually, sir I have pushed the wrong file after I have checked the file it was wrong so have deleted the file but actually it deleted the whole branch and automatically unassigned me .

**Comment by siddharth25pandey** [comment]
> > > @siddharth25pandey one more favor please could you reassigned to me
> > 
> > 
> > @soumik2012 It was assigned earlier to you. But you had removed from your choice by own.
> > Anyways, I'm going to assign you, but Please give a brief explanation why did you do so ?
> 
> Actually, sir I have pushed the wrong file after I have checked the file it was wrong so have deleted the file but actually it deleted the whole branch and automatically unassigned me .

@soumik2012 Alright ! no worries

**Comment by HarshCasper** [comment]
Please make an issue here and raise a PR, for an article as part of our Docs repository: https://github.com/TesseractCoding/NeoAlgo-Docs

It would be much appreciated. Thanks!

**Pull request merged**